src/app/accounts/create-account.ts

```diff
@@ -35,10 +42,17 @@ const initializeCreatedAccount = async ({
     enabledWallets[currency] = wallet
   }
 
-  // Set default wallet explicitly as BTC, or implicitly as 1st element in
+  // If enabled, randomly assign a default wallet based on the user id in order to A/B test the default wallet
+  const desiredDefaultWalletCurrency =
+    isUserIdEven(account.kratosUserId) || !config.randomizeDefaultWallet
```

Comment:
Why reliance on userId? A Math.random() > .5 function will work just as well and be more concise

Reply:
It allows for holistics to easily know the default wallet an account was created with. @jireva thought this would be the simplest way for constructing the dashboard, but potentially there are other ways.

Reply:
I believe that is the same as the current logic.